### PR TITLE
css

### PR DESCRIPTION
**Pull request by meinou**

### CODE REVIEW

**Comment by meinou** [comment]
@kirjs please review

File: apps/blog/src/app/feed/feed.component.scss [hunk 1]
```diff
@@ -39,12 +31,12 @@ ul {
   width: 100%;
 
   background: rgba(246,246,246,0.66);
```
Comment (kirjs) [inline apps/blog/src/app/feed/feed.component.scss]: I'd drop the gradient and use whatever material does

File: apps/blog/src/app/post/post.component.html [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-  post works!
-</p>
+
 <ng-container *ngIf="post$ | async as post">
-  <div>{{ post.title }}</div>
-  <div>{{ post.date }}</div>
-  <div>{{ post.author }}</div>
-  <div class="text" [innerHTML]="post.text"></div>
+    
+  <div> <markdown ngPreserveWhitespaces> # {{ post.title }}</markdown> by {{ post.author }}</div>
```
Comment (kirjs) [inline apps/blog/src/app/post/post.component.html]: You only need markdown for the text, not everywhere

**Comment by meinou** [comment]
@kirjs

File: apps/blog/src/app/feed/feed.component.html [hunk 3]
```diff
@@ -1,16 +1,21 @@
...
-    <li class="post" *ngFor="let post of posts$ | async">
+
+  <mat-card  *ngFor="let post of posts$ | async">
+    
+    <mat-card-content >
       <p class="head">
-        <span class="title">{{ post.title }}</span>
-        <span class="date">{{ post.date }}</span>
+        <span class="title">{{ post.title }}</span>      
+        <span class="date">by <span>{{ post.author }}</span> on {{ post.date | date: 'short' }}</span>
```
Comment (aziz512) [inline apps/blog/src/app/feed/feed.component.html]: Unnecessary span here: <span>{{ post.author }}</span>

File: apps/blog/src/app/feed/feed.component.scss [hunk 4]
```diff
@@ -21,30 +11,24 @@ ul {
...
+  padding-bottom: 10px;
 }
-
 .text {
   overflow: hidden;
-  height: 100px;
+  max-height: 100px;
+}
+
+.g {
```
Comment (aziz512) [inline apps/blog/src/app/feed/feed.component.scss]: give it a more descriptive name?

File: apps/blog/src/app/post/post.component.html [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
-  <div>{{ post.title }}</div>
-  <div>{{ post.date }}</div>
-  <div>{{ post.author }}</div>
-  <div class="text" [innerHTML]="post.text"></div>
+    
+  <div> {{ post.title }} by {{ post.author }}</div> 
+  <div> {{ post.date }}</div>
+  <div></div>
+  <markdown ngPreserveWhitespaces>{{ post.text }}</markdown>
+  <!-- <div class="text" [innerHTML]="post.text"></div> -->
```
Comment (aziz512) [inline apps/blog/src/app/post/post.component.html]: remove commented code (it'll be in git history anyway :) )

File: apps/blog/src/app/constants.ts [hunk 6]
```diff
@@ -0,0 +1,24 @@
+export const MARKDOWN_PLACEHOLDER = `
```
Comment (kirjs) [inline apps/blog/src/app/constants.ts]: Delete this?

**Comment by meinou** [comment]
@kirjs ?

**Pull request merged**